src/app/panels/text-panel/text-panel.component.html

```diff
@@ -8,6 +8,23 @@
             [active]="secondaryContent === 'INFO'"></evt-button>
         <evt-page-selector [pageID]="pageID" (selectionChange)="currentPage$.next($event)"></evt-page-selector>
         <evt-edition-level-selector [class.hidden]="hideEditionLevelSelector" [editionLevelID]="editionLevelID" (selectionChange)="currentEdLevel$.next($event)"></evt-edition-level-selector>
+        <ng-select
+            class="mr-1"
+            [closeOnSelect]="true" 
+            [searchable]="false"
+            bindLabel="id"
+            bindValue="id"
+            [items]="msDesc$ | async"
+            [clearable]="true"
+            (clear)="resetMsDesc();"
+            [placeholder]="'selectMsDesc' | translate" 
+            (change)="msDescID = $event.id; toggleMsDescContent()">
+            <ng-template
+                ng-id-tmp
+                let-item="item">
+                {{ item.id }}
```

Comment:
What if we don't have a specified id? I think we should avoid showing the xpath here. 
Ask Roberto for advice.